packages/libs/wallet-adapter-core/etc/wallet-adapter-core.api.md

```diff
@@ -52,6 +52,13 @@ export abstract class BaseWalletAdapter implements IAdapter {
     signTransaction(command: IUnsignedCommand | IUnsignedCommand[]): Promise<(IUnsignedCommand | ICommand) | (IUnsignedCommand | ICommand)[]>;
 }
 
+// @public (undocumented)
+export type CapabilityGuard = {
```

Comment:
Not that important, but maybe moving this code next to the other guards for easier mind mapping?